### PR TITLE
Changes on naming and small improvements

### LINKED ISSUE #26: Feedback on autogenerating new plugins
Here you have some feedback

- It is unclear that `plugin_name` and `module_name` are inserting at the beginning a string `nomad_` in the names. I'll suggest to simply delete this insertion, or use a more elaborated logic (I ended up too many times having a plugin called "nomad_nomad_...").
- The `github_username` used in the pyproject.toml to define the "Repository" field does not coincide with the place where the plugin is located, giving possible errors when releasing this package (or the need of anyways manually check). I'll suggest a more elaborated logic, or simply reading the real location of the repo (if this is possible...); the `github_username` should only write the authors/maintainers fields.
- The `description` of an entry point class in the `__init__.py` could perhaps be copied from the `description` of the plugin passed? It seems to me too generic to be useful, but I understand this might be not the point.
- For the future, it will be nice that the name of the classes of the parser/schema plugins can be passed. I don't know if this is possible, but it will save some much time.
- Lastly, I reiterate my point at #24. Specially, the `codeActionsOnSave` on the settings.json is very buggy. @blueraft, I can show you in private so you see how crazy Ruff gets on sorting imports. But independently of this, formatting should not be a barrier, and I am pretty much against this; I said this several times at the beginning when pylint was completely freezing computers due to the overload of checks and shits that was going on. And the solution was to simply delete those lines from `settings.json`.
- Also, a better explanation of the auto-generated README is needed. We did something for nomad-simulations, but maybe you can check: https://github.com/nomad-coe/nomad-simulations/blob/develop/README.md

### PR DESCRIPTION
**Pull request by JosePizarro3**
Closes #26 and #28 

@blueraft can you give it a look? Ideally, this should be merged asap, I just did some changes on naming and stuff.

Btw, if I add `example_uploads`, testing does not work because of not being able to import the `ExampleUploadsEntryPoint` class. But it is not relevant for these changes.

### CODE REVIEW

**Comment by JosePizarro3** [comment]
@lauri-codes do you mind checking on this? Ahmed is on holidays and I need this for the Hackathon

**Review by lauri-codes — COMMENTED** [review]
Here are some comments.

File: README.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ Cookiecutter prompts you for information regarding your plugin:
...
+module_name [nomad_awesome_tools]: nomad_awesome_tools
 short_description [NOMAD example template]: An awesome plugin for NOMAD
 version [0.1.0]:
 Select license:
 1 - MIT
 2 - BSD-3
 3 - GNU GPL v3.0+
 Choose from 1, 2, 3 [1]: 2
-include_schema_package [y/n] (y): y
+include_schema [y/n] (y): y
```
[the remark this answers is not in the document]
Comment (JosePizarro3) [reply README.md:34]: Is there any reason behind it? Now we are in a moment that is flexible enough to be changed back.

File: README.md [hunk 2]
```diff
@@ -54,13 +54,12 @@ nomad-awesome-tools/
...
-|       ├── schema_packages
+|       ├── schemas
 │       |   ├── __init__.py
-│       |   └── plugin.py
+│       |   └── schema.py
 |       └── parsers
 │           ├── __init__.py
-│           └── plugin.py
-|
+│           └── parsers.py
```
[the remark this answers is not in the document]
Comment (JosePizarro3) [reply README.md:62]: typo :/

File: README.md [hunk 3]
```diff
@@ -54,13 +54,12 @@ nomad-awesome-tools/
 ├── src
 │   └── nomad_awesome_tools
 │       ├── __init__.py
-|       ├── schema_packages
```
[the remark this answers is not in the document]
Comment (JosePizarro3) [reply README.md:57]: Ok, I saw this comment. I'll revert it

**Comment by JosePizarro3** [comment]
> Here are some comments.

Ok, done. I changed back to `schema_package(s)`.

**Comment by JosePizarro3** [comment]
@lauri-codes , you want to give another look or can I merge? I tested it, and all works: https://github.com/JosePizarro3/example-plugin/tree/pyscf-new-template

**Comment by lauri-codes** [comment]
@JosePizarro3: Looks good, go ahead and merge

**Comment by JosePizarro3** [comment]
Ah, I cannot merge. I don't have write access.

**Pull request merged**